src/agent/src/image_rpc.rs

```diff
@@ -207,7 +207,15 @@ impl ImageService {
     ) -> Result<String> {
         info!(sl(), "image metadata: {:?}", image_metadata);
         Self::set_proxy_env_vars();
-        if image_metadata["io.kubernetes.cri.container-type"] == "sandbox" {
+        let is_sandbox = if let Some(value) = image_metadata.get("io.kubernetes.cri.container-type") {
```

Comment:
It looks like the static check wants 
```suggestion
        let is_sandbox = if let Some(value) = image_metadata.get("io.kubernetes.cri.container-type")
        {
```
instead. I'm not sure why though as I think the current version is better and more consistent? I think my IDE would auto format that suggest to match what Julien added anyway, but maybe there is a refactor that would help it pass like:
```
let container_type = image_metadata.get("io.kubernetes.cri.container-type");
    let is_sandbox = if let Some(value) = container_type {
        value == "sandbox"
    } else if let Some(value) = image_metadata.get("io.kubernetes.cri-o.ContainerType") {
        value == "sandbox"
    } else {
        false
    };
```
?

Reply:
I see there are other "if let" statements here and there in the code where the accolade is on a newline, so I guess that's actually consistent - even if it seems to be an exception rather than the norm. Maybe it's a matter of line length too? I don't know.
Anyway, I went with the suggestion from "make check".